cookbook/controller/error_pages.rst

```diff
@@ -250,6 +250,127 @@ This approach allows you to create centralized and layered error handling:
 instead of catching (and handling) the same exceptions in various controllers
 time and again, you can have just one (or several) listeners deal with them.
 
+A centralized error handling that also allows to log exceptions might look like this::
+
+    // src/AppBundle/EventListener/AcmeExceptionListener.php
+    namespace AppBundle\EventListener;
+
+    use Psr\Log\LoggerInterface;
+    use Symfony\Component\HttpFoundation\Response;
+    use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
+    use Symfony\Component\HttpKernel\Exception\HttpExceptionInterface;
+
+    class AcmeExceptionListener
+    {
+        private $logger;
+
+        public function __construct(LoggerInterface $logger)
+        {
+            $this->logger = $logger;
+        }
+
+        public function onKernelException(GetResponseForExceptionEvent $event)
+        {
+            // Get the exception object from the received event
+            $exception = $event->getException();
+            $message = sprintf(
+                'Error: %s. Code: %s',
+                $exception->getMessage(),
+                $exception->getCode()
+            );
+
+            // Log the exception to the acme.log file
```

Comment:
"Log the exception to the configured log file (i.e. acme.log)" sounds better to me